src/service/action-impl.js

```diff
@@ -361,9 +361,9 @@ export class ActionService {
   /**
    * @param {!Element} node
    * @param {string} actionEventType
-   * @return {?ActionInfoDef}
+   * @return {?Array<ActionInfoDef>}
    */
-  matchActionInfo_(node, actionEventType) {
+  matchActionInfos_(node, actionEventType) {
```

Comment:
if we keep the array non-null, then `return actionMap[actionEventType] || null` in this method can just become `return actionMap[actionEventType] `